.github/workflows/publish_latest_fluvio.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+#  push:
+#    branches: [master]
+  workflow_dispatch:
+    inputs:
+      force:
+        required: false
+        description: 'Force push this release'
+
+env:
+  # Prod keys
+  # AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  # AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # Test keys
+  AWS_ACCESS_KEY_ID: ${{ secrets.PACKAGE_TEST_AWS_ACCESS_KEY_ID }}
```

Comment:
You can define 4 env: 2 for prod and 2 for test.  Then dynamically import env using `if`

Reply:
Please see this: https://github.community/t/accessing-secrets-by-index-using-an-environment-variable/17966/4